pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -1452,6 +1452,28 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 			Expect(len(causes)).To(Equal(1))
 			Expect(causes[0].Field).To(Equal("fake.domain.devices.interfaces[0].name"))
 		})
+		It("should reject a masquerade interface with a specified MAC address which is reserved by the BindMechanism", func() {
+			vmi := v1.NewMinimalVMI("testvm")
+			vmi.Spec.Domain.Devices.Interfaces = []v1.Interface{v1.Interface{
+				Name: "default",
+				InterfaceBindingMethod: v1.InterfaceBindingMethod{
+					Masquerade: &v1.InterfaceMasquerade{},
+				},
+				MacAddress: "02:00:00:00:00:00",
```

Comment:
should it us StaticMasqueradeBridgeMAC ?
(as it expected the error)

Reply:
Well, ref the constant would have explicitly mention why this is here and what is its purpose.
But I think you can achieve the same by using a named constant here as well to say the same, if you are looking to decouple this part from the production constant.

It is a nit, I will not insist on this.